Automattic/jetpack
# Social | Add disconnect/reconnect UI and logic

## Proposed changes:

* Wire up Disconnect button
* Add Reconnect button for broken connections

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:







You can switch between the new and old UI with `define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );
` in your wp-config file

  * MERGED - Ensure that your site points to your sandbox with D146900-code
  * Go to the social admin page
  *  Confirm that you see the Disconnect buttons for the connections
  * Click on Disconnect
  * Confirm that the connection is disconnected
  * Confirm that you see the notice for the same
  * Reload the page
  * Confirm that the disconnected connection is not in the list


https://github.com/Automattic/jetpack/assets/18226415/b1a8a1be-afbd-4e03-9964-a9f2f32eeccd


<img width="775" alt="Screenshot 2024-05-07 at 3 02 19 PM" src="https://github.com/Automattic/jetpack/assets/18226415/37e76ca6-dabf-437a-9dea-5e0526fc88d4">
<img width="791" alt="Screenshot 2024-05-07 at 3 03 35 PM" src="https://github.com/Automattic/jetpack/assets/18226415/df6475c0-1659-4da0-a393-605c28c4e04e">
<img width="742" alt="Screenshot 2024-05-07 at 3 04 15 PM" src="https://github.com/Automattic/jetpack/assets/18226415/a7bc6c10-5539-45f0-a9d2-fbec04d39cd1">
